_includes/configuration/paste_postprocess.md

```diff
@@ -0,0 +1,51 @@
+{% if page.title == "Paste plugin" %}
+### `paste_postprocess`
+
+This option enables you to modify the pasted content before it gets inserted into the editor but after it's been parsed into a DOM structure.
+
+**Type:** `Function`
+
+#### Example: Using `paste_postprocess`
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'paste',
+  menubar: 'edit',
+  toolbar: 'paste',
+  paste_postprocess: function(plugin, args) {
+    console.log(args.node);
+    args.node.setAttribute('id', '42');
+  }
+});
+```
+{% else %}
+### `paste_postprocess`
+
+This setting allows you to run custom filtering on the pasted content after it is run through PowerPaste's filters. The callback is passed two arguments: the PowerPaste plugin instance and an object containing event data. This object contains:
+
+- Standard paste event data.
+- `node` - A DOM node containing the DOM structure of the filtered paste content.
+- `mode` - A string indicating whether PowerPaste is in `clean`, `merge`, or `auto` mode.
+- `source` - A string indicating which kind of filtering PowerPaste will run based on the source of the content. This will return `html`, `msoffice`, `googledocs`, `image`, `imagedrop`, `plaintext`, `text`, or `invalid`.
+
+> **Note**: The `imagedrop` `source` was added in {{site.productname}} 5.3.
+
+> **Note**: The mode 'auto' is used when the content source does not have formatting to "clean" or "merge". For example, when pasting an image with no text or markup content.
+
+Example {{site.productname}} configuration:
```

Comment:
Should this be a heading?

Reply:
I'm going to say maybe and leave it as-is for now